site/en/docs/extensions/migrating/index.md

```diff
@@ -19,6 +19,24 @@ This section helps you upgrade an extension from Manifest V2 to Manifest V3, the
 
 We also have an [Extension Manifest Converter](https://github.com/GoogleChromeLabs/extension-manifest-converter). It does not do everything for you, but it will get you started. The converter's README describes what the tool changes.
 
-## New extension features
+## Keep the current set of features {: #keep-features }
```

Comment:
I know I suggested this, but seeing it here on the page it feels a bit random and out of context. How about adding a new section "Plan your deprecation" in which we can provide a high level summary:

* Analyse what needs to be migrated
* Avoid adding new (bigger) features during the migration
* Test your extension
* Gradually rollout

Wdyt?

Reply:
>How about adding a new section "Plan your deprecation" in which we can provide a high level summary:
> 
> * Analyse what needs to be migrated
> * Avoid adding new (bigger) features during the migration
> * Test your extension
> * Gradually rollout
> 
> Wdyt?
 
I'm not sure about the title "Plan your deprecation"... We need to think about it some more...

Would it go at the beginning or the end? Since the topics seem to combine before and after following the check list to convert...

<img width="620" alt="Screenshot 2023-09-15 at 16 37 00" src="https://github.com/GoogleChrome/developer.chrome.com/assets/37001393/7b90f3dd-142e-4d92-9322-8880b0fad6c6">

Reply:
> I'm not sure about the title "Plan your deprecation"
I'm not sure either...if, it should come first. But it depends on whether we have more items to add. If it's only the one I'm not sure it justifies a standalone section.